### PR TITLE
Subscription and SubAdmin bug fixes

### PR DESCRIPTION
**Pull request by giesler**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Disconnect-AzsPlanToOffer.ps1 [hunk 1]
```diff
@@ -19,46 +19,50 @@ Licensed under the MIT License. See License.txt in the project root for license
 .PARAMETER PlanName
     Name of the plan.
 
-.PARAMETER ResourceGroupName
+.PARAMETER ResourceGroup
```
Comment (deathly809) [inline src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Disconnect-AzsPlanToOffer.ps1]: Should be ResourceGroupName

File: src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Connect-AzsPlanToOffer.ps1 [hunk 2]
```diff
@@ -19,46 +19,50 @@ Licensed under the MIT License. See License.txt in the project root for license
 .PARAMETER PlanName
     Name of the plan.
 
-.PARAMETER ResourceGroupName
```
Comment (deathly809) [inline src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Connect-AzsPlanToOffer.ps1]: Leave as ResourceGroupName

File: src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Connect-AzsPlanToOffer.ps1 [hunk 3]
```diff
@@ -19,46 +19,50 @@ Licensed under the MIT License. See License.txt in the project root for license
...
         [string]
         $PlanName,
 
         [Parameter(Mandatory = $true, ParameterSetName = 'Offers_Link')]
         [System.String]
         $OfferName,
-
+    
         [Parameter(Mandatory = $true, ParameterSetName = 'Offers_Link')]
-        [ValidateLength(1, 90)]
```
Comment (deathly809) [inline src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Connect-AzsPlanToOffer.ps1]: Why was this deleted?

File: src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Connect-AzsPlanToOffer.ps1 [hunk 4]
```diff
@@ -19,46 +19,50 @@ Licensed under the MIT License. See License.txt in the project root for license
...
 
-    Links a plan to an offer.
+.EXAMPLE
+    Add-AzsPlanToOffer -PlanLinkType Addon -Offer offer1 -PlanName plan1 -ResourceGroupName rg1 -MaxAcquisitionCount 2
 #>
 function Add-AzsPlanToOffer
 {
     [CmdletBinding(DefaultParameterSetName='Offers_Link')]
-    param(
+    [CmdletBinding(SupportsShouldProcess = $true)]
```
Comment (deathly809) [inline src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Connect-AzsPlanToOffer.ps1]: This has to be a part of the other CmdletBinding, cannot do two CmdletBindings.

File: src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Disconnect-AzsPlanToOffer.ps1 [hunk 5]
```diff
@@ -19,46 +19,50 @@ Licensed under the MIT License. See License.txt in the project root for license
...
 
-    Unlink a plan from an offer.
+.EXAMPLE
+    Remove-AzsPlanToOffer -Offer offer1 -PlanName plan1 -ResourceGroup rg1
 #>
 function Remove-AzsPlanFromOffer
 {
     [CmdletBinding(DefaultParameterSetName='Offers_Unlink')]
-    param(
+    [CmdletBinding(SupportsShouldProcess = $true)]
```
Comment (deathly809) [inline src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Disconnect-AzsPlanToOffer.ps1]: Merge cmdlet bindings into one.

File: src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Connect-AzsPlanToOffer.ps1 (hunk 3 above)
Comment (giesler) [reply src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Connect-AzsPlanToOffer.ps1]: It was an unresolved merge conflict
